policy/release/rhtap_multi_ci/rhtap_multi_ci.rego

```diff
@@ -0,0 +1,41 @@
+#
+# METADATA
+# title: RHTAP Multi-CI
+# description: >-
+#   Checks for images built using an RHTAP build pipeline in either Jenkins,
+#   GitLab or GitHub. RHTAP pipelines are defined under
+#   https://github.com/redhat-appstudio/tssc-sample-templates/tree/main/skeleton/ci
+#
+package rhtap_multi_ci
+
+import rego.v1
+
+import data.lib
+
+# METADATA
+# title: RHTAP Multi-CI SLSA Provenance Attestation Found
+# description: >-
+#   Verify an attestation created by the RHTAP Multi-CI build pipeline is present.
+# custom:
+#   short_name: attestation_found
+#   failure_msg: "A SLSA v1.0 provenance with one of the following RHTAP Multi-CI
+#     build types was not found: %s."
+#   solution: >-
+#     It appears the build pipeline did not create the expected SLSA provenance
+#     attestation. Check for relevant error messages in the 'cosign-sign-attest'
+#     pipeline step logs.
+#   collections:
+#   - rhtap-multi-ci
+#   # For compatibility. We'll remove these soon.
+#   - rhtap-github
+#   - rhtap-gitlab
+#   - rhtap-jenkins
+#
+deny contains result if {
+	count(_attestations) < 1
+	result := lib.result_helper(rego.metadata.chain(), [lib.quoted_values_string(_known_build_types)])
+}
+
+_attestations := lib.rhtap_attestations
```

Comment:
I'd love to see a policy rule that applies to all slsa provenance attestations regardless of the CI system (beyond RHTAP). It could, optionally, filter attestations based on some criteria that is driven purely by policy rule data. For example:

```yaml
rule_data:
  slsa_provenance_build_types:
      - https://redhat.com/rhtap/slsa-build-types/jenkins-build/v1
      - https://redhat.com/rhtap/slsa-build-types/gitlab-build/v1
```

```rego
_attestations contains att if {
  some att in input.attestations
  # Ok to hard-code this I think since it should be v1 going forward.
  att.statement.predicateType == slsa_provenance_predicate_type_v1
  some build_type in lib.rule_data("slsa_provenance_build_types")
  att.statement.predicate.buildDefinition.buildType == build_type
}  
```

We could use that today in Konflux as well if we wanted to.